venue/views.py

```diff
@@ -2701,6 +2702,84 @@ def send_emails_with_referral_code(request):
         )
 
 
+ASSIGN_VERTO_ADDRESS_SCHEMA = AutoSchema(
```

Comment:
So that they can do this through the wallet. Why make this more complex than it needs to be.

Reply:
You can call through the wallet with `/api/authenticate` and pass the username and password. There are no restrictions on this call.

Going through a normal authentication flow does not make things more complex. It makes things less easy, I agree (because you need two steps vs. one). But it is less complex, because there is a standard flow for all API calls to go through.